### ISSUE TITLE
Windows Authentication support

### ISSUE THREAD

**Issue by kamazheng** [fullstackhero/blazor-starter-kit#298]
Is it possible to add windows authentication support?

**Comment by supermun** [comment]
I'm using LDAP to login.

This is a simple example:
[https://www.codeproject.com/Tips/5273877/Csharp-Validate-a-username-and-password-from-LDAP](url)

**Comment by mikecasas** [comment]
@supermun Are you having users login by entering their credentials?

**Comment by supermun** [comment]
What do you mean _"login by entering their credentials"_? We're using AD password to login.

**Comment by mikecasas** [comment]
@supermun  Do you have a textbox where the end user enters their AD password?

**Comment by neozhu** [comment]
If you are using your client PC inside the enterprise, you should not need to re-enter the username/password to verify that you are already logged in AD

**Comment by supermun** [comment]
> @supermun Do you have a textbox where the end user enters their AD password?

Yes, I'm using this login page: https://github.com/blazorhero/CleanArchitecture/blob/8120baac13afbf0711d8e7255ed72c336e8a570a/src/Client/Pages/Authentication/Login.razor

**Comment by mikecasas** [comment]
@supermun Thanks, that's what I was trying to confirm.

@neozhu That is the solution I was wanting to pursue.
